.github/workflows/manual-deploy-l1network.yml

```diff
@@ -58,15 +56,16 @@ jobs:
         uses: 'azure/aci-deploy@v1'
         with:
           resource-group: ${{ secrets.RESOURCE_GROUP }}
-          dns-name-label: testnet-obscuro${{ github.run_number }}
+          dns-name-label: testnet-gethnetwork-${{ github.run_number }}
           image: ${{ secrets.REGISTRY_LOGIN_SERVER }}/obscuronet/obscuro_testnet_l1network:latest
           registry-login-server: ${{ secrets.REGISTRY_LOGIN_SERVER }}
           registry-username: ${{ secrets.REGISTRY_USERNAME }}
           registry-password: ${{ secrets.REGISTRY_PASSWORD }}
-          name: testnet-app
+          name: testnet-gethnetwork-app
           location: 'uksouth'
           restart-policy: 'Never'
-          ports: '8025 8026 8027 9001 9002 9003'
+          command-line: /home/go-obscuro/integration/gethnetwork/main/main --numNodes=3 --startPort=8000 --websocketStartPort=9000 --prefundedAddrs=${{ secrets.GETHNETWORK_PREFUNDED_ADDR_WORKER }},${{ secrets.GETHNETWORK_PREFUNDED_ADDR_0 }},${{ secrets.GETHNETWORK_PREFUNDED_ADDR_1 }},${{ secrets.GETHNETWORK_PREFUNDED_ADDR_2 }}
+          ports: '8025 8026 8027 9000 9001 9002'
```

Comment:
@otherview Is it worth a comment here explaining why the Geth network starts on 8000, but our first port is 8025?

Reply:
yep 👍